cypress/integration/spec-pubsub.ts

```diff
@@ -0,0 +1,16 @@
+describe('The pubSub', () => {
```

Comment:
Remarque qui n'a rien à voir avec ce fichier, mais le dossier "integration" pourrait être renommé en "e2e" je pense comme on en avait parlé un peu vendredi :)

Reply:
I will make this change in an other branch.